app/controllers/people_controller.rb

```diff
@@ -68,7 +71,7 @@ def inactive
   end
 
   def index
-    @people = Person.active.select{|person| person.department&.name == 'CERT' || person.department&.name == 'Police'}
+    @people = Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
```

Comment:
How about
```
    Person.active.joins(:department).where("departments.manage_people": true)
```
If we're going to refactor it, let's get the hard coded strings out.